store/src/java/com/zimbra/cs/account/ldap/LdapLockoutPolicy.java

```diff
@@ -201,7 +206,28 @@ public void failedLogin(String protocol, String password) throws ServiceExceptio
         failedLogin(failedLogins, protocol, password);
     }
 
+    private void setCaptchaCount(String action ) {
+        Map<String, Object> attrs = new HashMap<String,Object>();
+        if(action.equals("update")) {
+            int loginFailCount = mAccount.getIntAttr(Provisioning.A_zimbraCAPTCHALoginFailedCount, 0);
+            attrs.put(Provisioning.A_zimbraCAPTCHALoginFailedCount, ++loginFailCount);
+        } else {
+            attrs.put(Provisioning.A_zimbraCAPTCHALoginFailedCount, 0);
+        }
+
+        try {
+            mProv.modifyAttrs(mAccount, attrs);
+        } catch (Exception e) {
+            ZimbraLog.account.warn("Unable to update account password lockout attrs: "+mAccount.getName(), e);
```

Comment:
Please use format style:
`ZimbraLog.account.warn("Unable to update account password lockout attrs: %s", mAccount.getName(), e);`
This removes un-necessary object creation when logging isn't enabled.